snuba/web/db_query.py

```diff
@@ -193,10 +181,11 @@ def execute_query_with_deduplication(
         query_id = md5(force_bytes(query.format_sql())).hexdigest()
         with state.deduper(query_id) as is_dupe:
             timer.mark("dedupe_wait")
-            stats.update({"is_duplicate": is_dupe, "query_id": query_id})
```

Comment:
By removing query_id from the stats the querylog_processor will not find the query_id anymore.
https://github.com/getsentry/snuba/blob/master/snuba/datasets/querylog_processor.py#L53

Reply:
This comment addresses this concern: https://github.com/getsentry/snuba/pull/901#discussion_r416126877

Reply:
oof, sorry about that.

Reply:
No problem, I wouldn't describe this as a particularly clear change. I think having the query encapsulate everything that goes into the query log (per https://github.com/getsentry/snuba/pull/901#discussion_r416153340) will make this easier to follow then when the query is a collection of otherwise unrelated mutable stack local variables.